.github/workflows/eslint.yml

```diff
@@ -4,6 +4,7 @@ name: eslint
 on: [push, pull_request]
 jobs:
   test:
+    runs-on: ubuntu-latest
```

Comment:
I agree that the code really looks "broken" without this line but how did you see an actual breakage?

Reply:
I got tons of emails

Reply:
The weird thing is that the check stopped to appear in the check list at the end of the PRs.

Now it is failing, but at least the file is valid again